src/pint/observatory/topo_obs.py

```diff
@@ -389,8 +416,8 @@ def _get_TDB_ephem(self, t, ephem):
         # Topocenter to Geocenter
         # Since earth velocity is not going to change a lot in 3ms. The
         # differences between TT and TDB can be ignored.
-        earth_pv = objPosVel_wrt_SSB("earth", t.tdb, ephem)
-        obs_geocenter_pv = gcrs_posvel_from_itrf(
+        earth_pv: PosVel = objPosVel_wrt_SSB("earth", t.tdb, ephem)
```

Comment:
What is the advantage to typing of this sort?  Is this to prevent any changes to the API?

Reply:
This tells the type checker to signal a problem if the function does not actually return a PosVel, or if the surrounding code uses the value in a way incompatible with a PosVel. More, it informs the reader what type this has, in case they don't know off the top of their head what type to expect. In this case it seemed useful because I wasn't ready to go digging about and annotate those two functions.

Reply:
ok, that makes sense.